daffodil-test/src/test/scala/org/apache/daffodil/section07/escapeScheme/TestEscapeScheme.scala

```diff
@@ -104,43 +104,10 @@ class TestEscapeScheme {
-  @Test def test_scenario3_3(): Unit = { runner2.runOneTest("scenario3_3") }
-  @Test def test_scenario3_4(): Unit = { runner2.runOneTest("scenario3_4") }
-  @Test def test_scenario3_5(): Unit = { runner2.runOneTest("scenario3_5") }
-  @Test def test_scenario3_6(): Unit = { runner2.runOneTest("scenario3_6") }
-  @Test def test_scenario3_7(): Unit = { runner2.runOneTest("scenario3_7") }
-  @Test def test_scenario3_8(): Unit = { runner2.runOneTest("scenario3_8") }
-  @Test def test_scenario3_9(): Unit = { runner2.runOneTest("scenario3_9") }
-  @Test def test_scenario3_10(): Unit = { runner2.runOneTest("scenario3_10") }
-  @Test def test_scenario3_10_postfix(): Unit = { runner2.runOneTest("scenario3_10_postfix") }
-  @Test def test_scenario3_11(): Unit = { runner2.runOneTest("scenario3_11") }
-  //DFDL-961
-  //@Test def test_scenario3_11_postfix() { runner2.runOneTest("scenario3_11_postfix") }
-  @Test def test_scenario3_12(): Unit = { runner2.runOneTest("scenario3_12") }
-  @Test def test_scenario3_12_req_term(): Unit = { runner2.runOneTest("scenario3_12_req_term") }
-  @Test def test_scenario3_13(): Unit = { runner2.runOneTest("scenario3_13") }
-  @Test def test_scenario3_13_postfix(): Unit = { runner2.runOneTest("scenario3_13_postfix") }
-  @Test def test_scenario3_14(): Unit = { runner2.runOneTest("scenario3_14") }
-  @Test def test_scenario3_14_req_term(): Unit = { runner2.runOneTest("scenario3_14_req_term") }
 
-  @Test def test_scenario4_1(): Unit = { runner2.runOneTest("scenario4_1") }
-  @Test def test_scenario4_2(): Unit = { runner2.runOneTest("scenario4_2") }
-  @Test def test_scenario4_3(): Unit = { runner2.runOneTest("scenario4_3") }
-  @Test def test_scenario4_4(): Unit = { runner2.runOneTest("scenario4_4") }
-  @Test def test_scenario4_5(): Unit = { runner2.runOneTest("scenario4_5") }
-  @Test def test_scenario4_6(): Unit = { runner2.runOneTest("scenario4_6") }
-  @Test def test_scenario4_7(): Unit = { runner2.runOneTest("scenario4_7") }
   @Test def test_scenario4_7_req_term(): Unit = { runner2.runOneTest("scenario4_7_req_term") }
-  @Test def test_scenario4_8(): Unit = { runner2.runOneTest("scenario4_8") }
-  @Test def test_scenario4_8_postfix(): Unit = { runner2.runOneTest("scenario4_8_postfix") }
-  @Test def test_scenario4_9(): Unit = { runner2.runOneTest("scenario4_9") }
   @Test def test_scenario4_9_req_term(): Unit = { runner2.runOneTest("scenario4_9_req_term") }
-  @Test def test_scenario4_10(): Unit = { runner2.runOneTest("scenario4_10") }
   @Test def test_scenario4_10_req_term(): Unit = { runner2.runOneTest("scenario4_10_req_term") }
-  @Test def test_scenario4_11(): Unit = { runner2.runOneTest("scenario4_11") }
-  @Test def test_scenario4_11_postfix(): Unit = { runner2.runOneTest("scenario4_11_postfix") }
-  @Test def test_scenario4_12(): Unit = { runner2.runOneTest("scenario4_12") }
   @Test def test_scenario4_12_req_term(): Unit = { runner2.runOneTest("scenario4_12_req_term") }
 
```

Comment:
Have you tested against the a regression suite to make sure this doesn't break anything? I doubt any schemas run into this issue, but we supported it before so I guess it's possible?

Reply:
Just tested against the full dfdl-schemas repo and everything passes, with one exception of a single vCard test, which also fails on master and doesn't appear to be related to my changes.